inference-engine/src/readers/ir_reader/ie_ir_parser.hpp

```diff
@@ -183,10 +184,20 @@ class V10Parser : public IParser {
 
     class XmlDeserializer : public ngraph::AttributeVisitor {
```

Comment:
I think we can move this class to cpp. Previously we implemented this class in the header because we had a legacy code which used this class.

Reply:
Will do after fix. To split fix and refactor.